pa_config/rca_idle_master.conf

```diff
@@ -102,9 +102,19 @@
   "muted-deciders": [],
   "muted-actions": [],
 
+  // Default Values are added in the config specific file as well
+  // Modify them as well whenever changing the default values (DeciderConfig.java)
   "decider-config-settings": {
+    // Decreasing order of priority for the type of workload we can expect on the cluster.
+    "workload-type": {
+      "priority-order": ["ingest", "search"]
+    },
+    // Decreasing order of priority for the type of cache which is expected to be consumed more.
```

Comment:
Minor: Consider rewording to make the ordering more clear. It should be clear that the list goes from most used to least used cache. (Same for workload priority).

Reply:
done